--- a/build/config/compiler/BUILD.gn	2021-10-11 21:19:18.000000000 +0900
+++ b/build/config/compiler/BUILD.gn	2021-10-11 21:19:33.000000000 +0900
@@ -2414,7 +2414,7 @@
       # It is skipped in tsan and asan because enabling it causes some
       # formatting changes in the output which would require fixing bunches
       # of expectation regexps.
-      cflags += [ "-gdwarf-aranges" ]
+#      cflags += [ "-gdwarf-aranges" ]
     }
 
     if (is_apple) {
