--- a/build_overrides/build.gni	2021-10-12 08:50:30.503989137 +0900
+++ b/build_overrides/build.gni	2021-10-13 00:26:46.028101036 +0900
@@ -35,7 +35,7 @@
 # Use bundled hermetic Xcode installation maintainted by Chromium,
 # except for local iOS builds where it's unsupported.
 # Allow for mac cross compile on linux machines.
-if (host_os == "mac" || host_os == "linux") {
+if (host_os == "mac") {
   _result = exec_script("//build/mac/should_use_hermetic_xcode.py",
                         [ target_os ],
                         "value")
