--- a/build/config/ios/BUILD.gn	2021-11-07 13:51:12.000000000 +0900
+++ b/build/config/ios/BUILD.gn	2021-11-07 18:26:02.000000000 +0900
@@ -138,7 +138,7 @@
   # TODO(crbug.com/1223481): Temporarily use a different
   # libclang_rt.iossim.a for arm64 simulator builds. This can be
   # removed when an arm64 slice is added to upstream Clang.
-  if (target_environment == "simulator" && current_cpu == "arm64") {
+  if (!use_xcode_clang && target_environment == "simulator" && current_cpu == "arm64") {
     assert(xcode_version_int == 1300)
     ldflags += [
       "-lSystem",
